pkgs/applications/misc/slweb/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv
+, fetchgit
+, redo-apenwarr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "slweb";
+  version = "0.5";
+
+  src = fetchgit {
```

Comment:
Shouldn't this use `fetchFromSourcehut`?

Reply:
Oh ! I didn't know about this ! Done :)